src/ingest_validation_tools/directory-schemas/seqfish-v0.yaml

```
@@ -1,5 +1,65 @@
 files:
   -
-    pattern: 'TODO'
-    description: 'Directory structure not yet specified.'
-    required: True
\ No newline at end of file
+    pattern: 'codebook/channel_[0-9]+\.(tsv|csv)'
+    description: 'Barcodes.'
+    required: True
+    is_qa_qc: False    
+  -
+    example: 'HybCycle_10/MMStack_Pos12.ome.tif'
+    pattern: 'HybCycle_[0-9]+/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
```

```suggestion
    pattern: 'HybCycle_[0-9]+/MMStack_Pos[0-9]+\.ome\.tiff?'
```
Here and below, I'd prefer more concise... but not essential, if you think the `|` is more readable.

---

@mccalluc I did it this way because it is easier to interpret by our users. However, if there is any scenario where it may fail, then let me know and I can update it. Though, it should be clear by the example.

---

@mccalluc the file naming was discussed with @mruffalo and @ctcisar in a meeting with CalTech. CalTech locally validated with this schema the next step is uploading so that @ctcisar can look at an actual submission.

---

Yep, what you have will function the same as what I proposed. This is fine.